### PR TITLE
Added loader in login form

### LINKED ISSUE #120: When a user clicks login button no response is given in frontend before it is authenticated or not
**Is your feature request related to a problem? Please describe.**
A leader is a suitable choice to notify the user that he/she had actually pressed the login button and server-side validation is taking time (1 or 2 seconds)

### PR DESCRIPTION
**Pull request by ashwaniYDV**
Fixes #120 

Changes: [Add here what changes you made in this Pull Request and if possible provide links depicting the changes.]

Surge Deployment Link: https://pr-121-sahaayak.surge.sh

Screenshots for the change: [Add here screenshots showing the changes.]

ezgif com-video-to-gif

### CODE REVIEW

File: src/components/LoginModal/LoginModal.js [hunk 1]
```diff
@@ -95,6 +95,9 @@ class LoginModal extends Component {
           onOk={this.handleSubmit}
         >
           {msg ? <Alert message={msg} type="error" /> : null}
+          <div className="example" style={{display: 'flex', justifyContent: 'center'}}>
```
Comment (harsh-jindal) [inline src/components/LoginModal/LoginModal.js]: @ashwaniYDV Add this loader on login button. Once the user clicks on login "Login" should be replaced by "loading icon"

**Review by harsh-jindal — CHANGES_REQUESTED** [review]

File: src/components/LoginModal/LoginModal.js (hunk 1 above)
Comment (ashwaniYDV) [reply src/components/LoginModal/LoginModal.js]: @harsh-jindal Ok

Comment (ashwaniYDV) [reply src/components/LoginModal/LoginModal.js]: @harsh-jindal done

**Review by harsh-jindal — APPROVED** [review]

**Comment by harsh-jindal** [comment]
LGTM 🚀

**Pull request merged**